intro-to-tests/practice-intro-to-pytest.md

```diff
@@ -963,6 +1036,22 @@ class TestPython1(unittest.TestCase):
 ```
 
 ##### !end-tests
+##### !hint
+
+Read through the test. Determine:
+
+- Which line is the "Arrange" step?
+- What are the "Act" and "Assert" steps?
```

Comment:
For this problem, to me, the Arrange and Assert are what are combined rather than Act and Assert.